src/cc/torchdistx/deferred_init.h

```diff
@@ -22,7 +22,8 @@ namespace torchdistx {
 // Forces all newly-constructed tensors on the calling thread to be fake while
 // also recording all operations performed on them in memory. Such tensors can
 // later be materialized by calling `materializeTensor()`.
-TDX_API void enableDeferredInit(bool value);
+TDX_API void enterDeferredInit();
```

Comment:
do the comments above need to be updated?

Reply:
The comments now apply to `enterDeferredInit()` instead of `enabledDeferredInit()`. Let me know if it is not clear.